<launch>
  <node pkg="navigation" name="fake_wheel_encoder_publisher" type="fake_wheel_encoder_publisher.py">
    <rosparam param="dynamic">false</rosparam>
    <rosparam param="wheel_raduis">false</rosparam>
    <rosparam param="wheel_distance">false</rosparam>
  </node>
</launch>